contractcourt/htlc_success_resolver.go

```diff
@@ -260,12 +257,33 @@ func (h *htlcSuccessResolver) broadcastReSignedSuccessTx() (
 			)
 		}
 
+		// Calculate the budget for this sweep.
+		value := btcutil.Amount(
+			secondLevelInput.SignDesc().Output.Value,
+		)
+		budget := calculateBudget(
+			value, h.Budget.DeadlineHTLCRatio,
+			h.Budget.DeadlineHTLC,
+		)
+
+		// The deadline would be the CLTV in this HTLC output. If we
+		// are the initiator of this force close, with the default
+		// `IncomingBroadcastDelta`, it means we have 10 blocks left
+		// when going onchain. Given we need to mine one block to
+		// confirm the force close tx, and one more block to trigger
+		// the sweep, we have 8 blocks left to sweep the HTLC.
```

Comment:
why do we need the *additional* block to trigger the sweep? Is this due to `CSV 1`?

Reply:
Ok lets correct that then

Reply:
We need one block to confirm the FC, since we use CSV 1, then yeah it implicitly satisfies the timelock. Once the CSV is expired, the output will be offered to the sweeper as a pending input, then one more block triggers the sweeper to actually sweep it.